test/integration/query-interface.test.js

```diff
@@ -586,14 +586,15 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
           expect(table).to.not.have.property('lastName');
         });
       });
-
-      it('should be able to remove a column with a foreign key constraint', function() {
-        return this.queryInterface.removeColumn('users', 'manager').bind(this).then(function() {
-          return this.queryInterface.describeTable('users');
-        }).then(function(table) {
+      if(dialect !== 'mysql'){
```

Comment:
Without including a schema to getForeignKeyQuery, it can mistakenly return a foreign key from another database with the same column name and table name.  This happens when removeColumn is called in MySQL dialect, since mysql can't really drop columns if it has foreign key constraints. That test is explicitly for testing removeColumn without a schema.